roles/foreman/defaults/main.yaml

```diff
@@ -0,0 +1,4 @@
+---
+foreman_image: quay.io/evgeni/foreman-rpm:3.12
```

Comment:
This is built using https://github.com/evgeni/foreman-container/blob/devel/Containerfile

Reply:
Is that included in this PR?

Reply:
Nope

Reply:
Do you want to move that Containerfile into this repo into the container-images directory? Or do you have thoughts on how we should manage the container images we build? (dedicated repo for just them, individual git repos for each image, etc.)

Reply:
https://github.com/ehelms/foreman-quadlet/pull/20